openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -527,21 +527,16 @@ def run(self, *, dry=False, verbose=True,
         )
 
         #  b. Write out a PDB containing the subsampled hybrid state
-        A_atoms = {hybrid_factory.old_to_hybrid_atom_map[i]
-                   for i in ligand_mappings['old_mol_indices']}
-        B_atoms = {hybrid_factory.new_to_hybrid_atom_map[i]
-                   for i in ligand_mappings['new_mol_indices']}
-        A_unique = list(A_atoms - B_atoms)
-        AB_core = list(A_atoms & B_atoms)
-        B_unique = list(B_atoms - A_atoms)
+        A_unique = list(hybrid_factory._atom_classes['unique_old_atoms'])
```

Comment:
I'm going to circle back on a similar comment you made last time re: extra assignments - Why do the assignment? Could you not just do the list call in the `np.in1d`?

Reply:
sure, now it's a hideously long line that isn't easy to read

Reply:
You could just... break up the line? formatting is a thing